addons/project/views/project_milestone_views.xml

```diff
@@ -17,7 +17,12 @@
                                 groups="project.group_project_milestone"
                                 close="1"
                         >
-                            <field name="task_count" string="Tasks" widget="statinfo"/>
+                            <div class="o_form_field o_stat_info">
+                                <span class="o_stat_value">
+                                    <field name="task_count" widget="statinfo" nolabel="1"/>
+                                </span>
+                                <span class="o_stat_text">Tasks</span>
+                            </div>
```

Comment:
I think we could avoid removing the field in the arch because in a customization, someone could have a xpath to select the field like this: `//button/field[@name='task_count']` for instance.
And so to avoid any errors, we could do this instead:
```suggestion
                            <-- TODO: Remove me in master -->
                            <field name="task_count" string="Tasks" widget="statinfo" invisible="1"/>
                            <div class="o_form_field o_stat_info">
                                <span class="o_stat_value">
                                    <field name="task_count" widget="statinfo" nolabel="1"/>
                                </span>
                                <span class="o_stat_text">Tasks</span>
                            </div>
```
and in master we will remove the field. 🙂